### PR TITLE
Adding additional information for ADAS systems

### PR DESCRIPTION
**Pull request by erikbosch**

### CODE REVIEW

File: spec/ADAS/ADAS.vspec [hunk 1]
```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
   type: sensor
   description: Indicates if ESC is currently regulating vehicle stability. True = Engaged. False = Not Engaged.
 
+ESC.IsStrongCrossWindDetected:
+  datatype: boolean
+  type: sensor
+  description: Indicates if ESC system is detecting strong cross winds.
+               True = Strong Cross winds detected.
```
Comment (ppb2020) [inline spec/ADAS/ADAS.vspec:207]: Other descriptions provide information for False as well. Granted, this is a bit redundant, but not providing it seems inconsistent (see line 201 above).

File: spec/ADAS/ADAS.vspec [hunk 2]
```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
...
+               
+ESC.RoadFriction:
+  type: branch
+  description: Road friction values reported by ESC system.
```
Comment (ppb2020) [inline spec/ADAS/ADAS.vspec:211]: by ESC -> by the ESC

File: spec/ADAS/ADAS.vspec [hunk 3]
```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
...
+
+ESC.RoadFriction.MostProbable:
+  datatype: float
```
Comment (ppb2020) [inline spec/ADAS/ADAS.vspec:214]: Is this level of precision useful given this is a percentage? That is, would 99.111% be much different from 99%.

File: spec/ADAS/ADAS.vspec [hunk 4]
```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
...
+  type: sensor
+  unit: percent
+  min: 0
+  max: 100
+  description: Most probable road friction, as calculated by ESC system.
```
Comment (ppb2020) [inline spec/ADAS/ADAS.vspec:219]: by ESC -> by the ESC

Here an throughout.

File: spec/ADAS/ADAS.vspec (hunk 3 above)
Comment (erikbosch) [reply spec/ADAS/ADAS.vspec:214]: As always difficult to know what resolution that is needed for future use-cases, float proposed to allow for higher precision than int. Of course, the difference between 99.1% and 99% is not that big, but the relative difference between 0.4% and 0% is bigger. And as with all VSS-signals, the actual accuracy/precision used in transmission can vary between implementations.

**Comment by danielwilms** [comment]
decision to merge: 22/05/03

**Pull request merged**